app/models/category.rb

```diff
@@ -4,4 +4,6 @@ class Category < ApplicationRecord
   validates :description, presence: true
 
   belongs_to :subsection
+  has_many :videos
```

Comment:
Add specs for all the changes you made in the models ;)

Reply:
Specs added!